src/Settings.h

```diff
@@ -60,7 +60,12 @@ class Settings : public QObject
     Q_OBJECT;
 
    public:
-    Settings(QObject* parent = nullptr) : QObject(parent), mAudioTester(new AudioTester)
+    Settings(bool guiEnabled = false, QObject* parent = nullptr)
+        : QObject(parent)
+#ifndef NO_GUI
+        , mGuiEnabled(guiEnabled)
+#endif
```

Comment:
mGuiEnabled depends on the default initialization value for bools for nogui builds? I think it's easier to reason about it without the ifdef.

Reply:
It's just so that in no GUI builds there's no risk of setting the variable to a value that should not be used.

Reply:
So in the NO_GUI builds, it'll always be false, regardless of whether someone passes something they shouldn't to the constructor. That way I didn't have to put a whole bunch of #idefs elsewhere in the code.

Reply:
Alright